src/response.rs

```diff
@@ -44,7 +44,33 @@ impl fmt::Debug for Response {
 }
 
 impl Response {
-    /// Get the final `Url` of this response.
+    /// Get the final `Url` of this `Response`.
+    ///
+    /// ```rust
+    /// # extern crate reqwest;
+    /// # #[macro_use] extern crate error_chain;
+    /// # use reqwest::{Client, Url};
+    /// #
+    /// # error_chain! {
+    /// #     foreign_links {
+    /// #         Reqwest(reqwest::Error);
+    /// #         Url(reqwest::UrlError);
+    /// #     }
+    /// # }
+    /// #
+    /// # fn run() -> Result<()> {
+    /// let url = Url::parse("http://httpbin.org")?;
+    /// let client = Client::new()?;
+    /// let resp = client.get(url.clone())?.send()?;
+    /// assert_eq!(resp.url(), &url);
+    /// # Ok(())
+    /// # }
+    /// # fn main() {
```

Comment:
Ah jeez, didn't even cross my mind to not call the `run` functions! I'll clean this pr up and then do another to get rid of network calls in the rest of the doc tests.